server/src/main/java/org/elasticsearch/repositories/RepositoriesService.java

```diff
@@ -69,7 +69,15 @@
 import static org.elasticsearch.snapshots.SearchableSnapshotsSettings.SEARCHABLE_SNAPSHOTS_REPOSITORY_UUID_SETTING_KEY;
 
 /**
- * Service responsible for maintaining and providing access to snapshot repositories on nodes.
+ * Service responsible for maintaining and providing access to multiple repositories.
+ *
+ * The elected master creates new repositories on request and persists the repository information in the cluster state. The cluster state
```

Comment:
suggest linking to the class here:

```suggestion
 * The elected master creates new repositories on request and persists the {@link RepositoryMetadata} in the cluster state. The cluster state
```